KsApi/models/Backing.swift

```diff
@@ -75,7 +75,8 @@ extension Backing: Decodable {
     self.amount = try values.decode(Double.self, forKey: .amount)
     self.backer = try values.decodeIfPresent(User.self, forKey: .backer)
     self.backerId = try values.decode(Int.self, forKey: .backerId)
-    self.backerCompleted = try values.decodeIfPresent(Bool.self, forKey: .backerCompleted)
+    self.backerCompleted = try values
+      .decodeIfPresent(Int.self, forKey: .backerCompleted) != nil ? true : false
```

Comment:
Should we check if it equals 1 instead of nil?

Reply:
I think the existence of a value means `true` in this case but there's no need for the ternary this can just be:

```swift
self.backerCompleted = try values.decodeIfPresent(Int.self, forKey: .backerCompleted) != nil
```